build-package

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+cd $(dirname ${BASH_SOURCE[0]})
+
+# Some variables
+DEBVERSION=1.99.5-1
```

Comment:
I see. It sounds weird but if that's the way it is...
Then I would definitely keep it 1.5.0 only in master until 2.0.0 is official, and you can have a long-standing PR for the pre-release that you update with new prereleases if you need them yourself, maybe.
Does this make sense?